### PR TITLE
Add minutes for SR2020 Covid-19 meeting

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Comment by Adimote** [comment]
Feel free to merge this once someone approves!

File: SR2020/2020-03-16.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Jenny Fletcher
+- Peter Law
+### Trustees:
+- Jimmy Thompson
+- Diane Dowling
+- David Massey
+
+## Competition will be Postponed
+- AI(trustees): Contact the kit team about postponing the competition. (Discuss their plans, etc..)
+- Reading university told us they cannot host us in April, and will reschedule our booking to late june/early july without cost. They’ve given us a
```
Comment (PeterJCLaw) [inline SR2020/2020-03-16.md]:
```suggestion
- Reading university told us they cannot host us in April, and will reschedule our booking to late june/early july without cost. They’ve given us a few dates to choose from.
```

File: SR2020/2020-03-16.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - 2 dropped out because of coronavirus.
+        - How many dropped out because the kit team arrived late?
+        - The kit team was told to collect the kits, if we get a response from the teams we should tell them to stop.
+    - Most teams can attend in June/July according to the form we sent out.
+- When to reschedule to:
+    - Plan to send a form to teams and volunteers to ask when they’re free:
+        - We should consider hotel prices when we pick a date.
+    - Other teams that may not be able to travel.
+
+- AI(andybs) call premier inn to cancel the booking in the morning.
```
Comment (PeterJCLaw) [inline SR2020/2020-03-16.md]: Maybe also enquire about postponing? Not sure if that would be possible. My main thinking is whether we can preserve the rate already agreed for the rooms.

Comment (Adimote) [reply SR2020/2020-03-16.md]: We can't preserve the rate, they recalculate it every time. Also we'd be postponing to a currently unknown date which they almost certainly don't support.

**Pull request merged**